### PR TITLE
Don't run tests for stable features with enable-preview

### PR DESCRIPTION
**Pull request by akurtakov**
## What it does
Tests for https://openjdk.org/jeps/441 and https://openjdk.org/jeps/456 without --enable-preview as these are stable features.

## How to test
Tests succeed.

## Author checklist

- [x] I have thoroughly tested my changes
- [x] The change is following the [coding conventions](https://wiki.eclipse.org/Platform/How_to_Contribute#Coding_Conventions)
- [x] I have signed the [Eclipse Contributor Agreement (ECA)](https://www.eclipse.org/legal/ECA.php)

### CODE REVIEW

**Comment by akurtakov** [comment]
Is there anyone objecting to this change? If not I'll merge it soon.

File: org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/SwitchPatternTest21.java [hunk 1]
```diff
@@ -2,19 +2,12 @@
...
 
 import junit.framework.Test;
 
 public class SwitchPatternTest21 extends AbstractBatchCompilerTest {
 
-	private static String[] JAVAC_OPTIONS = new String[] { "--enable-preview" };
-
-	static {
-		//	TESTS_NAMES = new String [] { "testNaming" };
-	}
```
Comment (stephan-herrmann) [inline org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/SwitchPatternTest21.java:16]: this block helps running individual tests, so we may want to keep it :)

**Review by stephan-herrmann — COMMENTED** [review]
no concerns, just a nitpick

File: org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/SwitchPatternTest21.java (hunk 1 above)
Comment (akurtakov) [reply org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/SwitchPatternTest21.java]: Restored.

**Pull request merged**